includes/save.php

```diff
@@ -445,86 +445,82 @@
-                if ($update) {
-                    $old = in_array($old_meeting->day, ['0', '1', '2', '3', '4', '5', '6']) ? $tsml_days[$old_meeting->day] : __('Appointment', '12-step-meeting-list');
-                }
-                $new = in_array($_POST['day'], ['0', '1', '2', '3', '4', '5', '6']) ? $tsml_days[$_POST['day']] : __('Appointment', '12-step-meeting-list');
-            } elseif ($field == 'time') {
-                if ($update) {
-                    $old = empty($old_meeting->time) ? '' : tsml_format_time($old_meeting->time, '');
-                }
-                $new = empty($_POST['time']) ? '' : tsml_format_time($_POST['time'], '');
-            } elseif ($field == 'end_time') {
-                if ($update) {
-                    $old = empty($old_meeting->end_time) ? '' : tsml_format_time($old_meeting->end_time, '');
-                }
-                $new = empty($_POST['end_time']) ? '' : tsml_format_time($_POST['end_time'], '');
-            } elseif ($field == 'region') {
-                if ($term = get_term($_POST['region'], 'tsml_region')) {
-                    $new = $term->name;
-                }
-                if ($update && !empty($old_meeting->region)) {
-                    $old = $old_meeting->region;
-                }
-            } elseif ($field == 'district') {
-                if (!empty($_POST['district']) && ($term = get_term($_POST['district'], 'tsml_district'))) {
-                    $new = $term->name;
-                }
-                if ($update && !empty($old_meeting->district)) {
-                    $old = $old_meeting->district;
-                }
-            } else {
-                if ($update) {
-                    $old = $old_meeting->{$field};
-                }
-                $new = $_POST[$field];
-            }
-
-            $field_name = __(ucwords(str_replace('_', ' ', $field)), '12-step-meeting-list');
-
-            if (in_array($field, $changes)) {
-                $message .= '<tr style="border:1px solid #999;background-color:#fff;"><td style="width:150px;padding:5px">' . $field_name . '</td><td style="padding:5px">';
-                if (!empty($old)) {
-                    $message .= '<strike style="color:#999">' . $old . '</strike> ';
-                }
-                $message .= $new . '</td></tr>';
-            } elseif (!empty($old)) {
-                $message .= '<tr style="border:1px solid #999;background-color:#eee;"><td style="width:150px;padding:5px">' . $field_name . '</td><td style="padding:5px">' . $old . '</td></tr>';
-            }
-        }
-        $message .= '</table>';
-        $subject = $update ? __('Meeting Change Notification', '12-step-meeting-list') : __('New Meeting Notification', '12-step-meeting-list');
-        $subject .= ': ' . sanitize_text_field($_POST['post_title']);
-        tsml_email($tsml_notification_addresses, $subject, $message);
-    }
+		if ($_POST['post_status'] !== 'draft') {
```

Comment:
could we do `($_POST['post_status'] === 'publish')` and include it with the if above? that way the diff should be smaller

Reply:
yep!

Reply:
more parens also didn't do it ... `if ((count($tsml_notification_addresses)) && (count($changes)) && ($_POST['post_status'] === 'publish')) {`

When 3 conditions are on the same line, I'm getting emails for drafts.

Reply:
i'd try `die($_POST['post_status'])` to see what's going on…